src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
-<html lang="en">
+<html
+  class="page"
+  lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/index.css">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body
+   class="page__body">
+    <div class="product-card">
+      <a
+        href="#"
+        class="product-card__link">
+        <img
+          src="./images/imac.jpeg"
+          alt="divice"
+          class="product-card__link__img"
+          >
```

Comment:
```suggestion
        <img
          src="./images/imac.jpeg"
          alt="divice"
          class="product-card__link__img"
        >
```
indentation for closing bracket - `>`